cryodrgn/protocols/protocol_base.py

```diff
@@ -230,12 +237,11 @@ def _createParticleSet(self):
         Create a set of particles with the associated z_values
         :return: a set of particles
         """
-        cryoDRGParticles = self.inputParticles.get()
-        ImgSet = self.getProject().getProtocol(cryoDRGParticles.getObjParentId()).inputParticles.get()
+        inputImgSet = self.inputParticles.ptcls.getExtended()
```

Comment:
I don0t think you need to use getExtended, just regular get() will give you the set (using internally the extended?

Reply:
are you saying I need to use inputImgSet = self.inputParticles.get().ptcls.get() ??